tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -344,11 +344,19 @@ Status IrEmitter::EmitAtomicOperationUsingCAS(const HloComputation& computation,
         IntToPtr(atomic_memory_address, atomic_address_type);
     binop_output_address =
         Add(PtrToInt(cas_new_output_address, address_int_type), offset);
-    binop_output_address = IntToPtr(binop_output_address, element_address_type);
+    binop_output_address = IntToPtr(
```

Comment:
@jerryyin can you please remove the unused variable `element_address_type` , it is throwing error internally , 
`third_party/tensorflow/compiler/xla/service/gpu/ir_emitter.cc:311:15: error: unused variable 'element_address_type' [-Werror,-Wunused-variable]
  llvm::Type* element_address_type = element_type->getPointerTo();
              ^
1 error generated.

third_party/tensorflow/compiler/xla/service/gpu/ir_emitter.cc:311:15: error: unused variable 'element_address_type' [-Werror,-Wunused-variable]
  llvm::Type* element_address_type = element_type->getPointerTo();
              ^
1 error generated.`

Reply:
Thanks, done. Did you manually add `-Werror`, `-Wunused-variable` for the compilation? Trying to figure out how to avoid this in the future.

Reply:
no copied error from internal logs

Reply:
@jerryyin This is another piece of infrastructure which is not working quite right, even though internally we build with `-Werror`, the external BUILD configuration is built with `-w`, which ignores all the errors.

Reply:
Thanks, I will add `-Werror` manually in the future.